GsaGH/Helpers/_SqlReader.cs

```diff
@@ -188,5 +190,58 @@ public static List<string> GetSectionsDataFromSQLite(List<int> type_numbers, str
 
       return section;
     }
+
```

Comment:
that´s copied from somewhere I am assuming?

Reply:
yes from composgh